requirements/requirements.in

```diff
@@ -26,6 +26,5 @@ raiden-contracts
 raiden-webui
 requests
 structlog
-typing-extensions
 web3
-webargs
+webargs
```

Comment:
need to add newline?

Reply:
That's probably just the work of his editor doing this automatically.

Reply:
mh, fixed it. I should configure PyCHarm to add this automatically